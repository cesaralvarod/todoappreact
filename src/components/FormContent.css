.form-content {
  width: 100%;
}

@media (min-width: 700px) {
  .form-content {
    width: 40%;
  }
}
