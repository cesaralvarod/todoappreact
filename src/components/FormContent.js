import React, { memo } from "react";

// Component

import TaskForm from "./forms/TaskForm";

// Styles

import "./FormContent.css";

const FormContent = memo(({ setArrData }) => {
  return (
    <div className="form-content">
      <h2>Add Task</h2>
      <TaskForm setArrData={setArrData} />
    </div>
  );
});

export default FormContent;
