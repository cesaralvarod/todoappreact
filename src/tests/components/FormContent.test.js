import React from "react";
import { shallow } from "enzyme";
import FormContent from "../../components/FormContent";

describe("Testing in component FormContent", () => {
  test("Component FormContent should show correctly", () => {
    const wrapper = shallow(<FormContent />);

    expect(wrapper).toMatchSnapshot();

    expect(wrapper.find("TaskForm").exists()).toBeTruthy();
  });
});
