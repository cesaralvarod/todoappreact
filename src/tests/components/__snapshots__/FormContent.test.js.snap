// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing in component FormContent Component FormContent should show correctly 1`] = `
<div
  className="form-content"
>
  <h2>
    Add Task
  </h2>
  <TaskForm />
</div>
`;
